ethereum/core/src/main/java/tech/pegasys/pantheon/ethereum/vm/MessageFrame.java

```diff
@@ -912,6 +920,11 @@ public Builder contract(final Address contract) {
       return this;
     }
 
+    public Builder contractAccountVersion(final int contractAccountVersion) {
+      this.contractAccountVersion = contractAccountVersion;
```

Comment:
Probably worth asserting the value is `>= 0`.

Reply:
done